locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Community Committee
+
+The Community Committee (CommComm) is a top-level committee in the Node.js Foundation focused on community-facing efforts. Code commits !== the only way to contribute!
+
+The CommComm has authority over community outreach including:
+ - Community Evangelism
+ - Education Initiatives
+ - Cultural Direction of Node.js Foundation
+ - Community Organization Outreach
+ - Project Moderation/Mediation
+ - Public Outreach and [Publications](https://medium.com/the-node-js-collection)
+
+There are four levels of engagement with the Community Committee:
+
+ 1. A **Contributor** is any individual creating or commenting on an issue or pull request.
+ 2. A **Collaborator** is a contributor who has been given write access to the repository
+ 3. An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+ 4. A **Member** is a collaborator with voting rights who has met the requirements of participation and voted in by the CommComm voting process.
```

Comment:
This seems a bit misleading to me, but maybe I've got some misconception.
This seems to say that there's some sort of hierarchy between those levels. But, unless I'm wrong, I see two different kind of involvments. Contributor/Collaborator on one side, and Observer/Member on the other side. For instance, you don't have to be Collaborator to become an Observer AFAIK, but this seems to say so.

Maybe at least turn this into a bullet list?

Reply:
I agree. Lets switch to a bullet list.

Reply:
TIL that "Collaborator" is an official class of participation in the CommComm, as defined in our Charter. We've strayed from that, mostly using Contributor, Observer, and Member - so we may need to go update the Charter in a bit here 😅

I do think the organization right now is good, since it's the general flow of how you'd go from being a collaborator to a member, but definitely +1 to making it bullets instead of a list.